server/app/views/applicant/ApplicantProgramBlockEditTemplate.html

```diff
@@ -1,13 +1,20 @@
 <!DOCTYPE html>
 <html>
     <head th:replace="~{applicant/ApplicantBaseFragment :: pageHeader}"></head>
-    <body>
+    <!-- TODO: This data attribute is waited for in screenshot tests. Update once we have modal dialogs -->
```

Comment:
**suggestion:** One of these todos have your username - consider making these consistent to make it easier to find later?

Reply:
Austin once reminded me that if the TODO is going to live for a while it's a good idea to create an issue and include the issue number

Reply:
Yeah that's even better :)

Reply:
done. This is the motivation I needed to finally make an epic and start making issues in github for tasks :)